scripts/qa_crowbarsetup.sh

```diff
@@ -659,8 +659,8 @@ EOF
     fi
 
     if [ -n "${localreposdir_target}" ]; then
-        for x in `seq 1 10` ; do
-            zypper rr 1
+        while zypper rr 1 2>&1 | grep -v "^Repository '1' not found"; do
```

Comment:
better use LANG=C zypper to ensure that you get the expected string - otherwise this becomes a busy endless loop, which is a downside to Dirk's and the previous version